pkg/sql/repository/CveStoreRepository.go

```diff
@@ -4,19 +4,34 @@ import (
 	"github.com/devtron-labs/image-scanner/pkg/sql/bean"
 	"github.com/go-pg/pg"
 	"go.uber.org/zap"
+	"time"
 )
 
 type CveStore struct {
 	tableName        struct{}      `sql:"cve_store" pg:",discard_unknown_columns"`
 	Name             string        `sql:"name,pk"`
 	Severity         bean.Severity `sql:"severity,notnull"`
-	Package          string        `sql:"package,notnull"`
+	Package          string        `sql:"package,notnull"` // deprecated, storing package data in image_scan_execution_result table
 	Version          string        `sql:"version,notnull"`
 	FixedVersion     string        `sql:"fixed_version,notnull"`
 	StandardSeverity bean.Severity `sql:"standard_severity,notnull"`
 	AuditLog
 }
 
+func (cve *CveStore) CreateAuditLog(userId int32) {
+	cve.CreatedBy = userId
+	cve.CreatedOn = time.Now()
+	cve.UpdatedBy = userId
+	cve.UpdatedOn = time.Now()
+}
+
+func (cve *CveStore) UpdateNewSeverityInCveStore(severity bean.Severity, userId int32) {
+	cve.Severity = severity
+	cve.StandardSeverity = severity
```

Comment:
what is the difference?

Reply:
I was also not clear with the diff btw these two, just that we were storing severity in two diff cols I did that, also I've seen the output of clair and trivy scanned reports and there was also no mention of standard severity, so need to understand this with og code owners as to why we are maintaining severity in two cols.